opentracing-shim/src/main/java/openconsensus/opentracingshim/SpanShim.java

```diff
@@ -125,4 +129,31 @@ public void finish(long finishMicros) {
     // TODO: Take finishMicros into account
     span.end();
   }
+
+  static Map<String, AttributeValue> convertToAttributes(Map<String, ?> fields) {
+    Map<String, AttributeValue> attrMap = new HashMap<String, AttributeValue>();
+
+    for (Map.Entry<String, ?> entry : fields.entrySet()) {
+      String key = entry.getKey();
+      Object value = entry.getValue();
+
+      // TODO - verify null values are NOT allowed.
+      if (value == null) {
+        continue;
+      }
+
+      if (value instanceof Short || value instanceof Integer || value instanceof Long) {
```

Comment:
`Byte`?

Reply:
Not sure we do get `Byte` values from OT at all (actually, the one we use mostly is `Integer`). Wondering if I should add it just in case ;)

Reply:
I just commenting on completeness =). I'm fine either way